python/src/main/python/drivers/piglit-worker.py

```diff
@@ -248,34 +256,63 @@ def run_image_job(frag_file: str, json_file: str, status_file: str,
     shader_test_file = graphicsfuzz_piglit_converter.get_shader_test_from_job(json_file)
 
     try:
-        runspv.log('Creating shader_test file...')
+        gfuzz_common.log('Creating shader_test file...')
         graphicsfuzz_piglit_converter.main_helper(arglist)
     except Exception as ex:
-        runspv.log('Could not create shader_test from the given job.')
+        gfuzz_common.log('Could not create shader_test from the given job.')
         raise ex
-
-    status = STATUS_SUCCESS
-
-    shader_runner_cmd_list = shader_runner_cmd() + [shader_test_file, SHADER_RUNNER_ARG_AUTO,
-                                                    SHADER_RUNNER_ARG_UNIFORMS,
-                                                    SHADER_RUNNER_ARG_SUBTESTS]
+    shader_runner_cmd_list = shader_runner_cmd() + \
+        [shader_test_file, SHADER_RUNNER_ARG_AUTO,
+         SHADER_RUNNER_ARG_UNIFORMS, SHADER_RUNNER_ARG_SUBTESTS]
+    if use_catchsegv:
```

Comment:
I would just define `use_catchsegv` as a local variable at the top of this function. Then do the exception thing (catch `Exception` instead of `BaseException` -- I think that should be fine). If we want to make things more flexible later, we can make `use_catchsegv` a parameter, but I doubt we will ever need to.

Reply:
I was worried about running tool_on_path for every image job, but it's probably no big deal in the long run.